app/bolt_listeners.py

```diff
@@ -317,12 +351,19 @@ def respond_to_new_message(
                 {
                     "content": f"<@{msg_user_id}>: "
                     + format_openai_message_content(
-                        reply.get("text"), TRANSLATE_MARKDOWN
+                        censor_string(input_string=reply.get("text")),
+                        TRANSLATE_MARKDOWN,
                     ),
                     "role": "user",
                 }
             )
 
+        latest_user_reply = ""
+        if messages[-1]["role"] == "user":
```

Comment:
Did you observe any negative effects with the user mentions here? If so, having some changes like this may be worth considering.

Reply:
I am using this to get message from users and store to DB but i am not merging here